pepper-apis/config/testing-inmemorydb.conf.ctmpl

```diff
@@ -216,7 +216,8 @@
     "somatic": {
         "realmToBucketMappings": [
             {"realm": "osteo2", "uploadBucket":"cmi-osteo2-uploaded-files"},
-            {"realm": "cmi-lms", "uploadBucket":"cmi-lms-uploaded-files"}
+            {"realm": "cmi-lms", "uploadBucket":"cmi-lms-uploaded-files"},
+            {"realm": "somatic-upload-test", "uploadBucket":"cmi-lms-uploaded-files"}
```

Comment:
Without this, the test fails because it can't find the upload bucket for the new test realm.